genesyscloud/integration_credential/resource_genesyscloud_integration_credential_test.go

```diff
@@ -161,6 +161,74 @@ func TestAccResourceCredential(t *testing.T) {
 	})
 }
 
+// Resource Credential DEVTOOLING-448
+// This tests to make sure that we can successfully create an integration credential for a Genesys Cloud oauth client without providing a client secret
+func TestAccGenesysCloudOAuthResourceCredential(t *testing.T) {
```

Comment:
This test case will attempt to create a Genesys Cloud OAuth Client and then create an integration credential with only a clientId.  If it creates successfully, the test should pass.